avatars/animationHelpers.js

```diff
@@ -189,7 +189,16 @@ async function loadAnimations() {
   const uint8Array = new Uint8Array(arrayBuffer);
   const animationsJson = zbdecode(uint8Array);
   animations = animationsJson.animations
-    .map(a => AnimationClip.parse(a));
+    .map(a => {
+      const animationClip = AnimationClip.parse(a)
+      if (animationClip.name === 'One Hand Sword Combo.fbx') {
+        trimClip(animationClip, 0.4, animationClip.duration - 0.5);
```

Comment:
This is not the place to fix animations. They should be fixed at bake time.

Reply:
Moved `trimClip()` from runtime to baker. https://github.com/webaverse/app/pull/2791/commits/f056c61d8d6daa123fee7d9c482dc0ed83211ada